django/template/base.py

```diff
@@ -201,6 +201,24 @@ def render(self, context):
         finally:
             context.render_context.pop()
 
+    def _stream(self, context):
+        return self.nodelist.stream(context)
+
+    def stream(self, context):
+        "Display stage -- can be called many times"
```

Comment:
I think it means it's where the actual display is created. As you said it's copied from the render method.